.github/workflows/tests.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       if: matrix.distro.name == 'debian'
       run: |
         apt-get update -q
-        apt-get install -qy gcc make linux-headers-amd64 linux-image-amd64 openssl
+        apt-get install -qy make linux-headers-amd64 linux-image-amd64 openssl
```

Comment:
Interesting - Arch and Fedora (IIRC) don't pull the compiler stack, when the headers are installed.

Should we do the same change in the Ubuntu case below?

Reply:
Already tried, the gcc dependency is required at least for older Ubuntu releases.

Reply:
Ack, please mention that in the commit message. For non Debian/Ubuntu experts such variations are non-obvious. Thanks

Reply:
_jfyi_ @xnox is the above variation, something Ubuntu team is willing to address? I don't have a strong opinion, just making sure devs in both distros are aware.